ext/standard/tests/file/disk_free_space_basic.phpt

```diff
@@ -4,6 +4,8 @@ Test disk_free_space and its alias diskfreespace() functions : basic functionali
 <?php
 if (getenv("TRAVIS") === "true") die("skip inaccurate on TravisCI");
 ?>
+--XFAIL--
+Some file systems (like BTRFS) have a fuzzy notion of "free space"
```

Comment:
See also https://bugs.php.net/80629. If the second value is never smaller than the first, maybe we should just check for `$space1 >= $space2`. Would that pass the test on your system?

Reply:
This works